.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: ci
+on: [push, pull_request]
+jobs:
+  # verify that project builds (via "check" via "clippy") on linux
+  linux-cargo-clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: clippy
+      - uses: actions-rs/cargo@v1
+        with:
+          command: clippy
```

Comment:
I would prefer not to add clippy: it's default profile has to many false positives to my taste, and I don't like silencing them. 

I have no strong feelings about this for this small repo, so, if you'd rather use clippy, I am ok with that!

Reply:
Fair enough. I agree about the false positives and the fact that may of the warnings are not very useful. I wouldn't mind if you decide to disable it, it's just part of the default ci configuration I use for new projects.